community/concepts/grain.md

```diff
@@ -0,0 +1,59 @@
+### Grain
+
+A big part of SourceCred’s vision is the desire to create project-specific cryptoassets, called "Grain" that represent contribution to or support of a project. These cryptoassets hold promise to make open-source projects financially sustainable which is currently an unsolved problem. You can read more about grain [here](concepts/grain.md).
+
+#### Properties of Grain
+
+ - Grain is always project-specific. For example, "SourceCred grain" and "Maker Community grain" would be wholly different assets.
+ - Every week, new grain is created, or "harvested" by the project.
+ - Most of the harvested grain goes to people that have directly worked on the project.
+ - A fraction of the harvested grain flows to those who hold grain in the project’s dependencies. For example, if you have grain in Python, then you would continually earn a small share of NumPy’s grain, and so forth.
+ - Grain is a freely transferrable and exchangeable asset. Most likely it will be implemented as an [ERC20 token](https://cointelegraph.com/explained/erc-20-tokens-explained), meaning that it can be bought or sold using the [0x Protocol 2](https://0x.org/).
```

Comment:
> meaning that it can be bought or sold using the [0x Protocol 2](https://0x.org/).

While this statement is true, it also implies that this is the only (or at least preferred) protocol to use when exchanging tokens. But actually, a competitor ([Uniswap](https://uniswap.io/)) has their own protocol, and is the most used in the crypto space right now. I wouldn't be surprised if ERC-20 tokens trade on Uniswap more than 0x, though they can trade on both. Perhaps we could say something like "...can be bought or sold on Decentralized Exchanges (DEX) using protocols such as [0x Protocol 2](https://0x.org/) and [Uniswap](https://uniswap.io/).

It's also possible that, should a project's Grain become valuable enough, it will trade on centralized exchanges, which use nether of these protocols.